packages/main/src/plugin/extension/local/extension-bundle.spec.ts

```
@@ -0,0 +1,244 @@
+      expect(folders[0]).toBe(join('path', 'extension1', 'builtin', 'extension1.cdix'));
+    });
+    test('recognizes an api extension when ext/package.json is not present', async () => {
+      readdirMock.mockResolvedValue([dirEntry]);
+      vi.mocked(existsSync)
+        // existSync on the folder path => true
+        .mockReturnValueOnce(true)
+        .mockReturnValueOnce(false);
+      const folders = await extensionBundle.readProductionFolders('path');
+
+      expect(folders).length(1);
+      expect(folders[0]).toBe(join('path', 'extension1', 'packages', 'extension', 'builtin', `extension1.cdix`));
```

Biome (2.5.6)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)